h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_7860_PUBDEV_7367_random_gridsearch_subspaces_TP_CS_binomial.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import division
+from __future__ import print_function
+from past.utils import old_div
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+from h2o.grid.grid_search import H2OGridSearch
+
+# I copied Karthik's test to perform gridsearch on TP/CS smoothers with random gridsearch.  
```

Comment:
Is this comment still relevant?

Reply:
Yes, this test is taken from Karthik and I just change the parameters and dataset.  I want to give him the credit and that is why the comment is there.